src/component/Search.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+// eslint-disable-next-line
+const Search = ({ onSearch }) => {
+  return (
+    <>
+      <div className="input">
+        <input type="search" onChange={onSearch} placeholder="Search exchange" />
+      </div>
+    </>
+
+  );
+};
+
+export default Search;
```

Comment:
- [ ] Kindly remove all `Eslint` disabling comments from this file. `Eslint` helps us to write clean and readable Javascript code. Using disabling comments defeats this purpose. 👍